src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,17 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (years) => {
```

Comment:
The indentation should be half of the current one on each line.

And I would suggest to use === instead of ==